src/lib/compiler/compileData.js

```diff
@@ -279,6 +279,28 @@ const compile = async (
     );
   });
 
+  // Replace ptrs in banked data
+  banked.mutate((data) => {
+    if(typeof data === "number") {
+      return data;
+    }
+    if(typeof data === "string" && data.startsWith("__REPLACE")) {
+      if(data.startsWith("__REPLACE:STRING_BANK:")) {
+        const index = parseInt(data.replace(/.*:/,''), 10);
+        return stringPtrs[index].bank;
+      }
+      if(data.startsWith("__REPLACE:STRING_HI:")) {
+        const index = parseInt(data.replace(/.*:/,''), 10);
+        return hi(stringPtrs[index].offset);
+      }
+      if(data.startsWith("__REPLACE:STRING_LO:")) {
+        const index = parseInt(data.replace(/.*:/,''), 10);
+        return lo(stringPtrs[index].offset);
+      }      
+    }
+    return 0;
```

Comment:
Not sure under what circumstances but I observed some cases where the `data` is of type `string` even if it's a number (you can see it happening in the bad build from numokeys if you `console.log(data)` when `typeof data === "string")` before the return statement).

To protect form those cases return could be `!isNaN(parseInt(data))?parseInt(data):0;` instead.